js/custom-menu.js

```diff
@@ -2,7 +2,7 @@
   Drupal.behaviors.exampleModule = {
     attach: function (context, settings) {
 
-      $('.cd-dropdown .cd-dropdown-toggle a').on('click', function(e){
+      $('.cd-dropdown .expanded a').on('click', function(e){
```

Comment:
Relates to similar change in styleguide.
This way, we can let Drupal output all its menu classes
https://github.com/UN-OCHA/styleguide/pull/22/commits/afb018ecf4f426c3b51bf9834049f0c428336b68